clippy_lints/src/methods/rewind_instead_of_seek_to_start.rs

```diff
@@ -0,0 +1,34 @@
+use clippy_utils::diagnostics::span_lint_and_then;
+use clippy_utils::match_qpath;
+use rustc_ast::ast::{LitIntType, LitKind};
+use rustc_errors::Applicability;
+use rustc_hir::{Expr, ExprKind};
+use rustc_lint::LateContext;
+use rustc_span::Span;
+
+use super::REWIND_INSTEAD_OF_SEEK_TO_START;
+
+pub(super) fn check<'tcx>(cx: &LateContext<'tcx>, expr: &'tcx Expr<'_>, arg: &'tcx Expr<'_>, name_span: Span) {
+    if_chain! {
+        if let ExprKind::Call(func, args1) = arg.kind;
+        if let ExprKind::Path(ref qpath1) = func.kind;
+        if match_qpath(qpath1, &["SeekFrom", "Start"]);
```

Comment:
Can `match_qpath` be replaced with `match_def_path`? The code below will be helpful:
https://github.com/rust-lang/rust-clippy/blob/502e87c3790059e878868125985726c065b72f85/clippy_lints/src/create_dir.rs#L39-L40